crates/cast/bin/main.rs

```diff
@@ -533,10 +533,7 @@ async fn main_args(args: CastArgs) -> Result<()> {
         }
         CastSubcommand::HashMessage { message } => {
             let message = stdin::unwrap_line(message)?;
-            let input = match message.strip_prefix("0x") {
-                Some(hex_str) => hex::decode(hex_str)?,
-                None => message.as_bytes().to_vec(),
-            };
+            let input = message.as_bytes().to_vec();
```

Comment:
as_bytes().to_vec() is not needed

Reply:
:+1: removed in https://github.com/foundry-rs/foundry/pull/9130/commits/942dc317918cbb99e5bd740a3177bdbd6a1d9772